$NetBSD: patch-ah,v 1.1 2013/03/16 21:32:01 dholland Exp $

For X11_TYPE=native, will have following problem without 
this patch:
|  Shared object "libpaper.so.1" not found

--- configure.orig	2012-08-08 17:04:57.000000000 +0900
+++ configure
@@ -5331,7 +5331,7 @@
 fi
 
 if test x$with_libpaper != xno; then
-    LIBS="$LIBS -lpaper"
+    LIBS="$LIBS -L${LPLIBDIR} -Wl,-R${LPLIBDIR} -lpaper"
 
 $as_echo "#define USE_LIBPAPER 1" >>confdefs.h
 
